src/host_beta/mainHost.ts

```diff
@@ -3,3 +3,7 @@
 
 import { MessageRouter } from './messageRouter';
 export const messageRouter = new MessageRouter(window);
+
+export function iframeFailed() {
```

Comment:
I'm not sure this function is necessary, I don't see it called anywhere?

Reply:
Oops! Forgot to remove. Good catch